PopPUNK/models.py

```diff
@@ -333,7 +334,7 @@ def fit(self, X, max_components):
         else:
             y = self.assign(self.subsampled_X, max_batch_size = self.max_batch_size)
         self.within_label = findWithinLabel(self.means, y)
-        self.between_label = findWithinLabel(self.means, y, 1)
+        self.between_label = findBetweenLabel_bgmm(self.means, y)
```

Comment:
Why this change in behaviour? This would change some of the examples I think

Reply:
This harmonises the behaviour of BGMM and DBSCAN - identifying the between-strain cluster as that containing the largest number of points. Name is a bit ugly.